app/assets/javascripts/components/shared/modals/modal-team.js

```diff
@@ -16,23 +16,27 @@
 
     initialize: function (options) {
       this.constructor.__super__.initialize.call(this, options);
+      this.data = options.data;
```

Comment:
For consistency sake, I would remove this declaration and add the `data` attribute to `defaults`.
You could also document it. What is the type/structure of `data`?